.changeset/chilled-ladybugs-push.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
+
+delta lake integration
```

Comment:
You can leave this one empty, practically it only contains change for `@finos/legend-manual-tests`

Reply:
done

Reply:
@aamend  by empty, he might the message shoudl be empty but it should still have `@finos/legend-manual-tests`

Reply:
see https://raw.githubusercontent.com/finos/legend-studio/master/.changeset/weak-mugs-shake.md

Reply:
This causes the changeset build to fail